### PR TITLE
Instead of returning an object with field `result` return the actual result

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Before result would look like:
`{"result": 42}`
Now the result looks like:
`42`

### CODE REVIEW

File: tests/near-bindgen/main_near.ts.expected [hunk 1]
```diff
@@ -165,17 +163,15 @@ export function rewrapFoobar(): void {
   let result = wrapped_rewrapFoobar(handler.__near_param_container);
 
   let encoder = new JSONEncoder();
-  encoder.pushObject(null);
 
   if (result != null) {
-    encoder.pushObject("result");
+    encoder.pushObject(null);
```
Comment (vgrichina) [inline tests/near-bindgen/main_near.ts.expected]: I think ideally we want to avoid dummy pushObjects

Comment (vgrichina) [reply tests/near-bindgen/main_near.ts.expected]: ah, I got it, it's not dummy, outer one was removed

**Comment by evgenykuzyakov** [comment]
The plan is 0.4.0 nearlib doesn't require result, while 0.3.5 does need `{"result":...}`.
For now I can just point templates to this commit and switch to 0.4.0

**Comment by evgenykuzyakov** [comment]
So 0.3.5 nearlib had some bugs. I fixed them and published 0.4.2 https://github.com/nearprotocol/nearcore/pull/773
Here is Studio https://github.com/nearprotocol/NEARStudio/pull/109

**Pull request merged**